### PR TITLE
Fix bugs in script import citations 

### PR DESCRIPTION
**Pull request by dglemos**

### CODE REVIEW

File: scripts/import/import_EPMC.pl [hunk 1]
```diff
@@ -744,7 +744,7 @@ sub clean_publications{
 
     # If there is publications without title, delete them 
     if($n_title_null != 0){
-      my $title_null_sth = $dba->dbc->prepare(qq[ select publication_id from publication where title is null or title = '' ]);
+      my $title_null_sth = $dba->dbc->prepare(qq[ select publication_id, pmid from publication where title is null or title = '' ]);
```
Comment (ima23) [inline scripts/import/import_EPMC.pl]: Where is the added pmid used?

File: scripts/import/import_EPMC.pl [hunk 2]
```diff
@@ -770,12 +781,10 @@ sub remove_publications{
 
     $var_id_sth->execute()||die;
     my $get_variation_ids = $var_id_sth->fetchall_arrayref();
-    my $variation_ids = $get_variation_ids->[0]->[0];
+    my $variation_ids = $get_variation_ids->[0];
```
Comment (ima23) [inline scripts/import/import_EPMC.pl]: Could a comment be added?

File: scripts/import/import_EPMC.pl (hunk 1 above)
Comment (dglemos) [reply scripts/import/import_EPMC.pl]: When removes the publication it write in a file the publication_id and in this case the tile is null so can't write the title, write the pmid instead. Writing only the publication_id is not info enough.

File: scripts/import/import_EPMC.pl (hunk 2 above)
Comment (dglemos) [reply scripts/import/import_EPMC.pl]: Done. Thanks.

**Comment by ima23** [comment]
merged to `release/99` and `master`

**Pull request merged**